backend/src/application/common/usecase.py

```diff
@@ -1,9 +1,15 @@
 from typing import Generic, TypeVar
+from fastapi import APIRouter, Response, status
+from ..infra/postgres/gateways import SAUserGateway
```

Comment:
Импорты у нас разделяются точками. При чём следи за тем, чтобы были абсолютные импорты относительно папки src

```suggestion
from src.infra.postgres.gateways import SAUserGateway
```